fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyticExpr.java

```diff
@@ -144,6 +145,11 @@ public AnalyticWindow getWindow() {
         return window;
     }
 
+    @Override
+    protected String getExprName() {
+        return Expression.normalizeColumnName(getFnCall().getExprName());
```

Comment:
this function should in `org.apache.doris.catalog.Column` or under org.apache.doris.common package

Reply:
I rethink this method, the actual mean of the method is normalize the name of expression or expr. and
it should be independent of column domain or other. So I think should rename it to normalizeName to match the actual meaning for normalize the expr(expression name). WDYT?